frontend/src/app/pages/login/login.component.ts

```diff
@@ -17,6 +17,7 @@ export class LoginComponent {
   public nurseForm: FormGroup;
 
   public loading: boolean = false;
+  public activeStep: 1 | 2 = 1;
```

Comment:
tuhle syntax jsem neznal. To má typ enum nebo je to nějaký zvláštní typ?

Reply:
myslim, že to vytvoří nový typ, jako bys napsal `typedef` a muže to být jen `1` nebo `2`

Reply:
typescript nepřestává překvapovat :D